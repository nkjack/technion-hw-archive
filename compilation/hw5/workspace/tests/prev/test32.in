void maxTestsStructInsideFunc() {
	int x;
	int x1;
	int y;
	int y1;
	byte t;
	byte t1;
	x = 15;
	y = 3 - 115;
	t = 33b;
	x1 = x;
	y1 = y;
	t1 =t;
	printi(x1);
	print("\n");
	printi(y1);
	print("\n");
	printi(t1);
}

void main() {
     maxTestsStructInsideFunc();
     print("\nI hate this course\n");
}
