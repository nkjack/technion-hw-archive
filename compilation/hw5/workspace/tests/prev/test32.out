Loaded: ./exceptions.s
15
-112
33
I hate this course
